### PR TITLE
change in the groovy script to adjust the service logo's.

### PR DESCRIPTION
**Pull request by ozancinar**

### CODE REVIEW

File: docs/service/createMarkdown.groovy [hunk 1]
```diff
@@ -58,7 +58,9 @@ if (dataFile != "template.json") {
     if (line.contains("%url%")) { line = process(line, data, "url") }
     if (line.contains("%doi%")) { line = processRich(line, data, "doi", "[https://doi.org/%doi%](https://doi.org/%doi%)") }
     if (line.contains("%description%")) { line = process(line, data, "description") }
-    if (line.contains("%screenshot%")) { line = processRich(line, data, "screenshot", "<img width=\"300\" align=\"right\" alt=\"screenshot of the service\" src=\"%screenshot%\">") }
+    // Here's the line added to make the service logo's consistent with the docs repo. The original line is still available (commented out) just below.
```
Comment (egonw) [inline docs/service/createMarkdown.groovy:61]:
```suggestion
```

File: docs/service/createMarkdown.groovy [hunk 2]
```diff
@@ -58,7 +58,9 @@ if (dataFile != "template.json") {
...
+    if (line.contains("%screenshot")) { line = processRich(line, data, "screenshot", "![screenshot of %service%](https://github.com/VHP4Safety/cloud/blob/main/docs/service/%screenshot% \"Click on the image to go to the service\")")}
+    //    if (line.contains("%screenshot%")) { line = processRich(line, data, "screenshot", "<img width=\"300\" align=\"right\" alt=\"screenshot of the service\" src=\"%screenshot%\">") }
```
Comment (egonw) [inline docs/service/createMarkdown.groovy:63]:
```suggestion
```

**Review by egonw — CHANGES_REQUESTED** [review]
The commit history already shows what changed. No need to keep the original line(s).

File: docs/service/createMarkdown.groovy (hunk 2 above)
Comment (ozancinar) [reply docs/service/createMarkdown.groovy:63]: Okay. I actually noticed a typo here; I think there should be a "%" after "screenshot" as well in the line.contains function.

**Pull request merged**